src/binary/ibc-setup/commands/balances.spec.ts

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs';
+import os from 'os';
+
+import test from 'ava';
+import sinon from 'sinon';
+
+import { IbcClient } from '../../../lib/ibcclient';
+import { generateMnemonic } from '../utils/generate-mnemonic';
+
+import { Options, run } from './balances';
+
+const consoleLog = sinon.stub(console, 'log');
+const fsReadFileSync = sinon.stub(fs, 'readFileSync');
+const ibcClient = sinon.stub(IbcClient, 'connectWithSigner');
+const mnemonic = generateMnemonic();
```

Comment:
Probably best to hardcode one here for reproducibility/transparency.

Reply:
done in 3ce03b0